website/static/js/share/stats.js

```diff
@@ -62,6 +62,9 @@ function timeGraph (data) {
         },
         legend: {
             show: false
+        },
+        tooltip: {
+          grouped: false
```

Comment:
Yep - involves writing some custom html too tho - as per our discussion, let's definitely do this post-beta. Here's an issue for later implementation - CenterForOpenScience/SHARE#152